KBS/evaluate_candidate.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Evaluating KBS Candidates
+
+  Scenario Outline: Project satisfies KBS requirements
+    Given a Project is active
+    And has an approval from <Approval Authority>
+    And is of Health Project Type <Type>
+#    And marked as not a candidate/last inclusion date is from a previous period
+    Then the Project should be listed as a KBS Candidate
+
+    Examples:
+      | Type                 | Approval Authority |
+      | Other clinical study | REK                |
+      | Other clinical study | CTIS               |
+      | Other clinical study | REK and CTIS       |
+      | Drug study           | REK                |
+      | Drug study           | CTIS               |
+      | Drug study           | REK and CTIS       |
+
+  Scenario: Notification of KBS Candidate
+    Given a user logged in as Project Manager
+    And at least one of their Projects is listed as a KBS Candidate
+    Then they are notified that they must decide if the Project is Reportable
```

Comment:
Should they be notified when the log in, or immediately? This is not clear from the scenario.

Two competing scenario interpretations:

```
    Given a User with the role of Project Manager for a Project
    And the Project is a Candidate for KBS reporting
    When the User logs in (where?)
    Then the User is notified that they must decide if their project is Reportable
    
    Given a User with the role of Project Manager for a Project
    When the Project becomes a Candidate for KBS reporting
    Then the User is notified that they must decide if their project is Reportable
```